app/validators/seat_availability_validator.rb

```diff
@@ -0,0 +1,9 @@
+class SeatAvailabilityValidator < ActiveModel::Validator
+  def validate(record)
+    status = record.seance.seats_hash[record.seat]
```

Comment:
Consider using better naming. `seats_hash` does not tell anything about real purpose of this method.
Maybe
```suggestion
    status = record.seance.seats_availability[record.seat]
```
WDYT?

Reply:
I agree, more descriptive:)